src/gui/Src/Gui/DataCopyDialog.cpp

```diff
@@ -16,23 +16,23 @@ DataCopyDialog::DataCopyDialog(const QVector<byte_t>* data, QWidget* parent) : Q
     mTypes[DataCWord] = FormatType { tr("C-Style WORD (Hex)"), 12 };
     mTypes[DataCDword] = FormatType { tr("C-Style DWORD (Hex)"), 8 };
     mTypes[DataCQword] = FormatType { tr("C-Style QWORD (Hex)"), 4 };
-    mTypes[DataCString] = FormatType { tr("C-Style String"), -1 };

```

Comment:
Why are these changes made?

Reply:
Because the UI has changed to prevent user from setting items per line to 0.